web/api/servers.go

```diff
@@ -1075,9 +1295,16 @@ func proxyHttpRequest(c *gin.Context, path string, ns *services.Node, node *mode
 
 	defer utils.CloseResponse(callResponse)
 
+	newHeaders := cleanHttpReturnErrors(callResponse.Header)
+
+	c.DataFromReader(callResponse.StatusCode, callResponse.ContentLength, callResponse.Header.Get("Content-Type"), callResponse.Body, newHeaders)
+	c.Abort()
```

Comment:
Should not need an abort, this should be the last op

Reply:
fixed in [3afa64c](https://github.com/pufferpanel/pufferpanel/pull/1379/commits/3afa64cabbf2a66b294418fec744d28d5024bb52)